src/frontend/js/data/useSession/index.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { createContext, PropsWithChildren, useCallback, useContext, useState } from 'react';
```

Comment:
You should add a module comment explaining what is this hook, why we need it, what you are doing, etc.